Automattic/wp-calypso
# Untangling: Add Marketing -> Connections

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9612

## Proposed Changes

This PR adds the Marketing -> Connections subtab under the `untangling/hosting-menu` flag.

<img width="1063" alt="image" src="https://github.com/user-attachments/assets/fdbc8bff-e6fa-46ea-b64c-4ec257551202">

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click any site
3. Go to Marketing -> Connections subtab.
4. Verify that this page is working.
5. Regression testing:
    - Go to /marketing/connections/:siteSlug of that site
    - Verify that this page is working.

## Pre-merge Checklist
